src/components/approval-type/ApprovalType.stories.ts

```diff
@@ -0,0 +1,37 @@
+import { ApprovalType } from './index'
+
+export default {
+  title: 'component/ApprovalType',
+  component: ApprovalType,
+  argTypes: {
+  }
```

Comment:
Super nit picky, but probably doesn't need a new line. ie `argtypes: {}`

Reply:
Nice, I'll fix it!